source/configuration_manual/forwarding_parameters.rst

```diff
@@ -1,74 +1,85 @@
+Dovecot supports proxying various pieces of information and even
+variables for various protocols when forwarding connection. It requires
+that the sender is listed under ``trusted_networks``. For IMAP, it uses
+the ``ID`` command, for other protocols, ``XCLIENT`` is used.
 
-This feature is supported since v1.2, except for x-forward which was added in v2.2.29.
+This feature is supported since v1.2, except for x-forward which was
+added in v2.2.29.
 
 IMAP protocol
-=============
+-------------
 
-For IMAP protocol, this is done by extending the ID command. The maximum length of parameters to parse is 255 bytes. The parameters are forwarded as part of the ID command key-value list.
+For IMAP protocol, this is done by extending the ID command. The maximum
+length of parameters to parse is 255 bytes. The parameters are forwarded
+as part of the ID command key-value list.
 
 ``5 ID (x-originating-ip "127.0.0.1" x-originating-port "143"... )``
 
 Supported parameters
-====================
+~~~~~~~~~~~~~~~~~~~~
 
-* ``x-originating-ip`` - Client IP
+-  ``x-originating-ip`` - Client IP
 
-* ``x-originating-port`` - Client port
+-  ``x-originating-port`` - Client port
 
-* ``x-connected-ip`` - Server IP
+-  ``x-connected-ip`` - Server IP
 
-* ``x-connected-port`` - Server port
+-  ``x-connected-port`` - Server port
 
-* ``x-proxy-ttl`` - TTL which is reduced by each hop, loop prevention. When TTL drops to 0, the connection is dropped.
+-  ``x-proxy-ttl`` - TTL which is reduced by each hop, loop prevention.
+   When TTL drops to 0, the connection is dropped.
 
-* ``x-session-id / x-session-ext-id`` - Session ID to be used.
+-  ``x-session-id`` / ``x-session-ext-id`` - Session ID to be used.
 
-* ``x-forward-<variable_name>``- Forwarded variable, see :ref:`config_variables`
+-  ``x-forward-<variable_name>`` - Forwarded variable, see
+   :ref:`Variables <config_variables>`
 
 POP3 protocol
-=============
+-------------
 
-For POP3 protocol, this is done with custom ``XCLIENT`` command which accepts a space separated list of ``key=value`` parameters.
+For POP3 protocol, this is done with custom ``XCLIENT`` command which
+accepts a space separated list of key=value parameters.
 
 Supported parameters
-====================
+~~~~~~~~~~~~~~~~~~~~
 
-* ``ADDR`` - Client IP
+-  ``ADDR`` - Client IP
 
-* ``PORT`` - Client port
+-  ``PORT`` - Client port
 
-* ``SESSION`` - Session ID
+-  ``SESSION`` - Session ID
 
-* ``TTL`` - TTL which is reduced by each hop, loop prevention. When TTL drops to 0, the connection is dropped.
+-  ``TTL`` - TTL which is reduced by each hop, loop prevention. When TTL
+   drops to 0, the connection is dropped.
 
-* ``FORWARD`` - Base64 encoded ``key=value`` parameter to be stored.
+-  ``FORWARD`` - Base64 encoded key=value parameter to be stored.
```

Comment:
Base64-encoded list of key=value pairs to be forwarded to auth process. The keys and values are escaped using Dovecot's tab-escape format.